serverlessworkflow/modules/ROOT/pages/cloud/operator/developing-workflows.adoc

```diff
@@ -1,8 +1,11 @@
-= Developing Workflow Services with the Operator
+= Developing Workflow Applications with the Operator
```

Comment:
```suggestion
= Developing Workflows with the Operator
```
Maybe we can just say workflows?  As especially with this single file approach, it's not really an application imho.  Also, for prod build chapter we also say "Building and Deploying Workflows with the Operator".